app/main.py

```diff
@@ -1,6 +1,34 @@
+import os
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.getenv("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print(f"Performing request to Weather API for city {FILTERING}...")
+
+    params = {
+        "key": API_KEY,
+        "q": FILTERING
+    }
+
+    result = requests.get(URL, params=params)
+
+    if result.status_code == 200:
+        result = result.json()
+        location = result["location"]
+        weather = result["current"]
+        data = (
+            f"{location['name']}/{location['country']} "
+            f"{location['localtime']} "
+            f"Weather: {weather['temp_c']} Celsius, "
+            f"{weather['condition']['text']}"
+        )
+        print(data)
+    else:
+        print("Invalid FILTERING OR API_KEY!!!")
```

Comment:
Error can be also on server side or on another place, you will show this error on every case, change it to be more abstract